assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -196,6 +188,18 @@ che.oauth.openshift.verify_token_url= NULL
 # a recurring schedule.
 schedule.core_pool_size=10
 
+# Name of cofig map in Che server namespace with additional CA TLS certificates to be propogated into all user's workspaces.
+# If the property is set on OpenShift 4 infrastructure, then cluster CA bundle will be propogated too.
+che.trusted_ca.bundles_configmap=NULL
```

Comment:
Are `bundles_configmap` and `workspace_bundle_configmap` named in different way (bundle & bundles) by purpose?

Reply:
I'm a bit confused. Is it for Che Server or Workspace, or both?

Reply:
> Are `bundles_configmap` and `workspace_bundle_configmap` named in different way (bundle & bundles) by purpose?

Yes, they are. `ca.bundles_configmap` contains several CA bundles (that's why `s`). `workspace_bundle_configmap` is a merged (from bundles above) bundle which is propagated inside workspaces via mount from single config map.

Reply:
> I'm a bit confused. Is it for Che Server or Workspace, or both?

Short answer - for both. `che.trusted_ca.bundles_configmap` is for Che server and is located in `che` namespace. `che.trusted_ca.workspace_bundle_configmap` is a copy of `che.trusted_ca.bundles_configmap` but in a workspace namespace.